masterlist.yaml

```
@@ -6384,6 +6384,11 @@ plugins:
     tag:
       - Delev
       - Relev
+    msg:
+      - type: say
+        content: 'It is recommended that you use [Diverse Skyrim - Battle of Whiterun Fix](https://www.nexusmods.com/skyrimspecialedition/mods/47939) with this mod.'
+        lang: eng
```

@PierreDespereaux The message can be modified to fit a boilerplate message that already exists, so the end result will be: "It's highly recommended that you also use [Diverse Skyrim - Battle of Whiterun Fix](https://www.nexusmods.com/skyrimspecialedition/mods/47939)."

Also, that language key shouldn't be there.

```suggestion
      - <<: *alsoUseX
        subs: [ '[Diverse Skyrim - Battle of Whiterun Fix](https://www.nexusmods.com/skyrimspecialedition/mods/47939)' ]
```